django_rq/views.py

```diff
@@ -58,6 +59,28 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
+def _collect_workers_per_configuration(queue_list):
+    """Collects, into a list, dictionaries of connections_config and its
+    workers
+    """
+    workers_collections = []
```

Comment:
Correct me see if I understood it wrong, but `connection_config` is a `dict`, which in an unhashable type, and I cannot use it as a key.